### DISCUSSION TITLE
How important is the report card?

### DISCUSSION THREAD

**Discussion by noah-de** [projectdiscovery/subfinder#631] [Q&A] [answered]
Seeing the impressive good grade on the report card in the project [readme](https://github.com/projectdiscovery/subfinder/blob/master/README.md), I clicked on it and reviewed [the results](https://goreportcard.com/report/github.com/projectdiscovery/subfinder). Seeing that the results were from 11 months ago, I clicked 'refresh now' and saw that:

    ' There was an error processing your request: Could not analyze the repository: no .go files found '

For the entered path:
github.com/projectdiscovery/subfinder

Adding the install path from the readme returns different errors:
github.com/projectdiscovery/subfinder/v2/cmd/subfinder@latest

Essentially, it is a false record until it is correct.
How can we get this corrected?

**Comment by forgedhallpass** [comment, marked as answer]
Hello @noah-de,

Thank you for pointing this out. The reason why you receive that message is because the tool has been changed and it does not look for go files in modules/subdirectories anymore. We'll fix this, but in the meantime, if you would like to see the correct values, you can add `/v2` to the end of the [URL](https://goreportcard.com/report/github.com/projectdiscovery/subfinder/v2).
